tock/hours/forms.py

```diff
@@ -74,6 +74,15 @@ def render_option(self, selected_choices, option_value, option_label):
             conditional_escape(force_text(option_label)))
 
 
+def choice_label_for_project(project):
+    """
+    Returns the label for a project as it should appear in an
+    auto-completable list of choices.
+    """
+
+    return '%s - %s' % (project.id, project.name)
```

Comment:
Note that I could have added this as a method to the `Project` class, but I didn't want to bloat that class, since this functionality seems fairly specific to forms (and the timecard form in particular).  I could move it over to the class if that's preferred, though.